Updating README file (#45)
Shopify/koa-shopify-auth: README.md

## Installation

> This package depends on the [Shopify Node Library](https://github.com/Shopify/shopify-node-api) being included in your app, so you should install both of them:

Reviewer: *should* install it? Isnt it already installed via dependency?

Author: So this is a bit weird, and I haven't had a chance to test this before we release `koa-shopify-auth` - if you use a local import you're not able to import the dependency directly, but I fully expect it to work with an actual package import. I didn't want to state that here until I had a chance to verify it.

);
```

### Migrating from cookie-based authentication to session tokens

Versions prior to v4 of this package used cookies to store session information for your app. However, internet browsers have been moving to block 3rd party cookies, which creates issues for embedded apps.

> If you have an app using this package, you can migrate to session tokens by performing a few steps:

Reviewer 2: ```suggestion
If you have an app using this package, you can migrate from cookie-based authentication to session tokens by performing a few steps:
```

Author: Thanks for the copy check, applying them all!

- Upgrade your `@shopify/koa-shopify-auth` dependency to v4+
- Install `@shopify/shopify-api` in your backend app

> - Update your server as per the [Usage](#usage) instructions to properly initialize the library and middleware

Reviewer 2: ```suggestion
- Update your server as per the [Usage](#usage) instructions to properly initialize the `@shopify/shopify-api` library and the `@shopify/koa-shopify-auth` middleware
```

- Install `@shopify/app-bridge-utils` in your frontend app

> - Replace `fetch` calls with `authenticatedFetch` from App Bridge Utils

Reviewer 2: ```suggestion
- In your frontend app, replace `fetch` calls with `authenticatedFetch` from App Bridge
```

## Gotchas

### Session

> The provided `MemorySessionStorage` class is not fit for production use. You can implement your own strategy by creating a class that implements a few key methods. Learn more about [how the Shopify Library handles sessions](https://github.com/Shopify/shopify-node-api/blob/main/docs/issues.md#notes-on-session-handling).

Reviewer 2: ```suggestion
The provided `MemorySessionStorage` class may not be scalable for production use. You can implement your own strategy by creating a class that implements a few key methods. Learn more about [how the Shopify Library handles sessions](https://github.com/Shopify/shopify-node-api/blob/main/docs/issues.md#notes-on-session-handling).
```
